ctapipe/irf/binning.py

```diff
@@ -0,0 +1,104 @@
+"""Collection of binning related functionality for the irf tools"""
+import astropy.units as u
+import numpy as np
+from pyirf.binning import create_bins_per_decade
+
+from ..core import Component
+from ..core.traits import Float, Integer
+
+
+def check_bins_in_range(bins, range):
+    low = bins >= range.min
+    hig = bins <= range.max
+
+    if not all(low & hig):
+        raise ValueError(f"Valid range is {range.min} to {range.max}, got {bins}")
+
+
+class OutputEnergyBinning(Component):
+    """Collects energy binning settings."""
+
+    true_energy_min = Float(
+        help="Minimum value for True Energy bins in TeV units",
+        default_value=0.005,
+    ).tag(config=True)
+
+    true_energy_max = Float(
+        help="Maximum value for True Energy bins in TeV units",
+        default_value=200,
+    ).tag(config=True)
+
+    true_energy_n_bins_per_decade = Float(
+        help="Number of edges per decade for True Energy bins",
+        default_value=10,
+    ).tag(config=True)
+
+    reco_energy_min = Float(
+        help="Minimum value for Reco Energy bins in TeV units",
+        default_value=0.015,
+    ).tag(config=True)
+
+    reco_energy_max = Float(
+        help="Maximum value for Reco Energy bins in TeV units",
+        default_value=200,
+    ).tag(config=True)
+
+    reco_energy_n_bins_per_decade = Float(
+        help="Number of edges per decade for Reco Energy bins",
```

Comment:
Name is `n_bins`, help says `n_edges`

Reply:
Value means it should be `n_bins` like the variable name, so the help needs to be updated.

Reply:
:+1:  Thats a problem at multiple places. Will fix.